components/camel-ftp/src/main/java/org/apache/camel/component/file/remote/FtpOperations.java

```diff
@@ -269,6 +269,9 @@ public boolean deleteFile(String name) throws GenericFileOperationFailedExceptio
     public boolean renameFile(String from, String to) throws GenericFileOperationFailedException {
         log.debug("Renaming file: {} to: {}", from, to);
         try {
+            //make use of the '/' separator because JSch expects this 
```

Comment:
JSch is only used for SftpOperations. So this comment is a bit mis-leading. ftp is using commons-net

Reply:
good catch. i will update soon.